.github/workflows/deb.yml

```diff
@@ -0,0 +1,89 @@
+name: Build package
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
```

Comment:
I don't think we want to build the deb on pull request. When the PR gets merged that triggers the push event, that's the only time we want the deb to be built.